### DISCUSSION TITLE
Possible combination with chatGPT?

### DISCUSSION THREAD

**Discussion by MineDead27Gamer** [Monika-After-Story/MonikaModDev#10210] [QA - code and content creation (technical)] [answered]
As title says. maybe there is a possibility to add chatGPT functions into the game, so we can chat with Monika more lively (no matter how strange it may sound)

**Comment by multimokia** [comment, marked as answer]
This would definitely be submod territory as chatGPT isn't perfect even now and can be prompt-engineered to break its usual boundaries. Overall we can't make sure it fits within our vision for Monika's personality. That and the API usage for this feature would be significant, and rather costly.
